dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/CompanyDataControllerTest.kt

```diff
@@ -217,23 +217,6 @@ class CompanyDataControllerTest {
         )
     }
 
-    @Test
```

Comment:
Why is this no longer necessary?

Reply:
This test is meant to ensure that frameworks that do not have a display-page are not considered when calculating available search filters. We do not have any frameworks without a view page and this functionality has already been removed in the backend in August. Therefore, I feel this test is not required. But I'm happy to discuss it.

Reply:
Agree with @MarcTM01

Reply:
Ok, then i will close this here